examples/api/test/widgets/undo_history/undo_history_controller.0_test.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/widgets/undo_history/undo_history_controller.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('The undo history controller should undo and redo the history changes', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const example.UndoHistoryControllerExampleApp(),
+    );
+
+    expect(find.byType(TextField), findsOne);
+    expect(find.widgetWithText(TextButton, 'Undo'), findsOne);
+    expect(find.widgetWithText(TextButton, 'Redo'), findsOne);
+
+    await tester.enterText(find.byType(TextField), '1st change');
+    await tester.pump(const Duration(milliseconds: 500));
```

Comment:
nit: is there a constant that we can use for this?

Reply:
The constant is private 😕

Reply:
Thanks for the link! I think you can make a similar constant directly in the test file with some comments, just so people know what the value 500ms signifies.

Reply:
What do you think of [refactor: Add a const for duration](https://github.com/flutter/flutter/pull/148205/commits/2578f8fd12b00f34825c6703a4219cb7cdd8de9a) ? Did you have a better comment in mind?